probe-rs/src/rtt.rs

```diff
@@ -250,8 +401,8 @@ impl Rtt {
                 tracing::debug!("Scanning region: {:?}", region);
 
                 vec![Range {
-                    start: region.start as u64,
-                    end: region.end as u64,
+                    start: region.start,
```

Comment:
Can this be a `region.clone()`?

Reply:
Thank you!
e8741b5ddd548c2744fee23597e7767b70285ec5